velox/functions/prestosql/aggregates/MapAggAggregate.cpp

```diff
@@ -185,8 +190,8 @@ class MapAggAggregate : public exec::Aggregate {
     return size;
   }
 
-  VectorPtr removeDuplicates(MapVectorPtr mapVector) const {
-    mapVector->canonicalize();
+  VectorPtr removeDuplicates(MapVectorPtr& mapVector) const {
+    MapVector::canonicalize(mapVector);
```

Comment:
This usage contradicts the comment on the canonicalize method: "Do not use except right after construction."